Resources/Prototypes/_EE/Body/Organs/xelthia.yml

```
@@ -0,0 +1,29 @@
+        maxVol: 50
+      food:
+        maxVol: 5
+        reagents:
+        - ReagentId: UncookedAnimalProteins
+          Quantity: 5
+  - type: Metabolizer
+    maxReagents: 3
+    metabolizerTypes: [ Xelthia ]
+    removeEmpty: true
+    groups:
+      - id: Food
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)